src/commands.rs

```diff
@@ -402,7 +410,22 @@ fn print_status(ctx: &mut Context<'_>, device: &nitrokey::DeviceWrapper<'_>) ->
 
 /// Inquire the status of the nitrokey.
 pub fn status(ctx: &mut Context<'_>) -> anyhow::Result<()> {
-  with_device(ctx, |ctx, device| print_status(ctx, &device))
+  let device_infos = find_devices(&ctx.config)?;
+  anyhow::ensure!(
+    !device_infos.is_empty(),
+    "Nitrokey device not found{}",
+    format_filter(&ctx.config)
+  );
+
+  let mut manager =
+    nitrokey::take().context("Failed to acquire access to Nitrokey device manager")?;
+  for (idx, device_info) in device_infos.iter().enumerate() {
+    let device = manager
+      .connect_path(device_info.path.deref())
+      .context("Failed to connect to Nitrokey")?;
```

Comment:
Would it make sense to include the path in the error here?

Reply:
Sure.